code/datums/inventory_slots/slots/slot_ears.dm

```diff
@@ -0,0 +1,35 @@
+/datum/inventory_slot/ear
+	slot_name = "Left Ear"
+	slot_id = slot_l_ear_str
+	slot_state = "ears"
+	ui_loc = ui_l_ear
+	covering_slot = slot_head_str
+	covering_flags = SLOT_EARS
+	can_be_hidden = TRUE
+	requires_organ_tag = BP_HEAD
+	requires_slot_flags = SLOT_EARS
+
+/datum/inventory_slot/ear/update_overlay(var/mob/living/user, var/obj/item/prop, var/redraw_mob = TRUE)
+	user.update_inv_ears(redraw_mob)
+
+/datum/inventory_slot/ear/get_examined_string(mob/owner, mob/user, distance, hideflags, decl/pronouns/pronouns)
+	if(_holding && !(hideflags & HIDEEARS))
+		return "[pronouns.He] [pronouns.has] [_holding.get_examine_line()] on [pronouns.his] [lowertext(slot_name)]."
+
+/datum/inventory_slot/ear/unequipped(var/mob/living/user, var/obj/item/prop, var/redraw_mob = TRUE)
+	. = ..()
+	if(.)
+		for(var/slot in global.ear_slots)
+			if(slot == slot_id)
+				continue
+			var/datum/inventory_slot/inv_slot = user.get_inventory_slot_datum(slot)
+			if(inv_slot?.get_equipped_item() == prop)
+				inv_slot.clear_slot()
+
+/datum/inventory_slot/ear/can_equip_to_slot(var/mob/user, var/obj/item/prop, var/slot, var/disable_warning, var/force)
+	return ..() && prop.w_class <= ITEM_SIZE_TINY
```

Comment:
This is why CI is failing. `/obj/item/clothing/head/hairflower`, used in the test, is not `ITEM_SIZE_TINY` and therefore can't be equipped now.
```
			if((w_class > ITEM_SIZE_TINY) && !(slot_flags & SLOT_EARS))
				return FALSE
```
It seems you've inverted the current check, which is intended to allow tiny items to be hidden in the ear slot even without the SLOT_EARS flag.
```suggestion
	return ..() || prop.w_class > ITEM_SIZE_TINY
```
I checked and couldn't find any other cases where this logic was inverted; pockets use something similar and it works fine.

Reply:
Gah, don't know how I messed the logic up that bad.